ipn/ipnlocal/local.go

```diff
@@ -2275,6 +2275,9 @@ func (b *LocalBackend) pingPeerAPI(ctx context.Context, ip netip.Addr) (peer *ta
 	if !ok {
 		return nil, "", fmt.Errorf("no peer found with Tailscale IP %v", ip)
 	}
+	if peer.Expired {
+		return nil, "", errors.New("peer has expired")
```

Comment:
"peer's key has expired" or something?

this is user-visible, so maybe @mayakacz has opinions. You can submit now and tweak later, but heads up to Maya.

Reply:
"peer's node key has expired" or "peer's node key is expired"

Reply:
Done, thanks @mayakacz!